cmd/sovereignnode/incomingHeader/incomingHeaderHandler.go

```diff
@@ -24,17 +24,17 @@ type ArgsIncomingHeaderHandler struct {
 	Hasher      hashing.Hasher
 }
 
-type incomingHeaderHandler struct {
+type incomingHeaderProcessor struct {
 	headersPool HeadersPool
 	txPool      TransactionPool
 	marshaller  marshal.Marshalizer
 	hasher      hashing.Hasher
 }
 
-// NewIncomingHeaderHandler creates an incoming header handler which should be able to receive incoming headers and events
+// NewIncomingHeaderProcessor creates an incoming header handler which should be able to receive incoming headers and events
```

Comment:
creates an incoming header **processor**

Reply:
Right, missed it